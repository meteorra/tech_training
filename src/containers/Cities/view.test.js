import React from 'react';
import renderer from 'react-test-renderer';
import CitiesView from './view';

function setup() {

    const props = {
        changeRoute: jest.fn(),
        deleteListItem: jest.fn(),
        cities: [],
    };

    const View = CitiesView(() => null);
    const render = renderer.create(<View {...props} />);

    return {
        render,
    };
}

describe('CitiesView', () => {

    it('renders h1 title "Discover the weather in Belarus cities"', () => {

        const { render, } = setup();
        expect(render.toJSON()).toMatchSnapshot();
    });
});
