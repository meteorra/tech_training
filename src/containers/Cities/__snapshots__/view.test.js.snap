// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesView renders h1 static title "Discover the weather in Belarus cities" 1`] = `
<div
  className="row"
>
  <div
    className="col-md-8 col-md-offset-2"
  >
    <div
      className="panel panel-default"
    >
      <div
        className="panel-body"
      >
        <h1>
          Discover the weather in Belarus cities
        </h1>
        <hr />
      </div>
    </div>
  </div>
</div>
`;
